src/e2e/tatum.token.spec.ts

```diff
@@ -0,0 +1,280 @@
+import { Ethereum } from '../dto'
+import { Network, TatumSDK } from '../service'
+
+describe('Tatum token', () => {
+  let tatum: TatumSDK<Ethereum>
+  beforeAll(async () => {
+    tatum = await TatumSDK.init<Ethereum>({
+      network: Network.ETHEREUM_SEPOLIA,
+      verbose: true,
+      retryDelay: 1000,
+      retryCount: 2,
+    })
+  })
+
+  describe('token balances', () => {
+    it('should get token balances', async () => {
+      const { data: balance } = await tatum.token.getBalance({
+        addresses: ['0x2cbaf358c0af93096bd820ce57c26f0b7c6ec7ab'],
+      })
+      expect(balance).toBeDefined()
+      expect(balance.length).toBe(2)
+      expect(balance[0]).toStrictEqual({
+        address: '0x2cbaf358c0af93096bd820ce57c26f0b7c6ec7ab',
+        balance: '100',
+        chain: 'ethereum-sepolia',
```

Comment:
can we use Network.ETHEREUM_SEPOLIA please? or it is not compatible?

Reply:
I would keep it explicit like this since we are calling core-api (sort of 3rd party) and the response can change, or the enum Network can change and then affect the tests.